irohad/torii/impl/command_service_impl.cpp

```diff
@@ -11,10 +11,7 @@
 #include "common/byteutils.hpp"
 #include "common/is_any.hpp"
 #include "common/visitor.hpp"
-#include "cryptography/default_hash_provider.hpp"
-#include "interfaces/iroha_internal/transaction_batch_factory.hpp"
-#include "interfaces/iroha_internal/transaction_sequence.hpp"
-#include "validators/default_validator.hpp"
+#include "interfaces/iroha_internal/transaction_batch.hpp"
```

Comment:
Is it needed? We do not get into batch in that class, after all

Reply:
https://github.com/hyperledger/iroha/blob/2ecdd677d0070c9a1e299ff7ca3b939de635bbb5/irohad/torii/impl/command_service_impl.cpp#L129